web/ui/react-app/src/Graph.tsx

```diff
@@ -242,21 +243,24 @@ class Graph extends PureComponent<GraphProps, GraphState> {
   };
 
   destroyPlot = () => {
-    this.$chart.destroy && this.$chart.destroy();
+    if (isPresent(this.$chart)) {
+      this.$chart.destroy && this.$chart.destroy();
```

Comment:
Do you still need the `this.$chart.destroy` check? Shouldn't `destroy` always exist on an existent chart?

Reply:
Nope....still sleeping :)